Dockerfile

```
@@ -33,3 +33,15 @@ COPY charts /charts
 WORKDIR /
 
 ENTRYPOINT ["/gardener-controller-manager"]
+
+############# scheduler #############
+FROM alpine:3.8 AS scheduler
+
+RUN apk add --update bash curl openvpn tzdata
```

```suggestion
RUN apk add --update bash curl
```

Why is openvpn/tzdata needed in the scheduler container?